metaflow/plugins/kfp/kfp_pod_decorator.py

```diff
@@ -0,0 +1,27 @@
+from metaflow.decorators import StepDecorator
+
+
+class PodAnnotationDecorator(StepDecorator):
+    """
+    Add a single pod annotation. This decorator can be used multiple time per step.
+    Repeated adding assigning value under same name will overwrite previous value.
+    """
+
+    name = "annotation"
+    defaults = {
+        "name": None,
+        "value": None,
+    }
+
+
+class PodLabelDecorator(StepDecorator):
+    """
+    Add a single pod label. This decorator can be used multiple time per step.
+    Repeated adding assigning value under same name will overwrite previous value.
+    """
+
+    name = "label"
```

Comment:
@alexlatchford @talebzeghmi did we want to keep the naming convention as `@kfp-label` and `@kfp-annotation`?

Reply:
`@kfp_label` and `@kfp_annotation` works fine for me.